src/Components/CardLIst/index.jsx

```diff
@@ -0,0 +1,140 @@
+        editModeFlag: false,
+        cardSelectionFlag: false,
+      },
+      {
+        id: '7',
+        title: 'Title 7',
+        text: 'It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.',
+        editModeFlag: false,
+        cardSelectionFlag: false,
+      },
+      {
+        id: '8',
+        title: 'Title 8',
+        text: 'Cicero, written in 45 BC. book is a treatise on the theory of ethics, very popular during the Renaissance. The first line of Lorem Ipsum, "Lorem ipsum dolor sit amet..", comes from a line in section 1.10.32.',
+        editModeFlag: false,
+        cardSelectionFlag: false,
+      },
+      {
+        id: '9',
+        title: 'Title 9',
+        text: 'Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for lorem ipsum will uncover many web sites still in their infancy.',
+        editModeFlag: false,
+        cardSelectionFlag: false,
+      },
+      {
+        id: '10',
+        title: 'Title 10',
+        text: 'by injected humour, or randomised words which look even slightly believable. If you are going to use a passage of Lorem Ipsum, you need to be sure there anything embarrassing hidden in the middle of text. All the Lorem Ipsum generators on the Internet tend to repeat predefined chunks as necessary',
+        editModeFlag: false,
+        cardSelectionFlag: false,
+      },
+    ],
+    viewModeFlag: false,
+  });
+
+  const saveNewChanges = (value) => {
+    const itemIndex = state.cards.findIndex((item => {
+      return item.id === value.id
+    }));
+    let card = { ...state.cards[itemIndex] };
+    card = { ...value };
+    const newCardState = [...state.cards];
+    newCardState[itemIndex] = card;
+    setstate({ cards: newCardState });
+  }
+
+  const viewHandler = () => {
+    const newCardsState = [];
+    [...state.cards].forEach(item => {
+      newCardsState.push({ ...item, editModeFlag: false });
+    });
+    setstate({
+      cards: newCardsState,
+      viewModeFlag: !state.viewModeFlag,
+    });
+  }
+
+  const deleteCards = () => {
+    const newCardsState = [];
+    [...state.cards].forEach(item => {
+      if (item.cardSelectionFlag === false) {
+        newCardsState.push(item)
+      }
+    })
+    setstate({
+      cards: newCardsState,
+    });
+  }
+
+  return (
+    <div>
+      <div className="nav_block">
+        <p><input className='view_mode_btn' onClick={viewHandler} type="checkbox" id='check1' />view only</p>
+        <p><button className='del_mode_btn' onClick={deleteCards}><AiFillDelete /></button>delete items</p>
+      </div>
```

Comment:
These items have nothing to do with CardList component. Please either move it to a header or create a separate component.

Reply:
Fixed